### ISSUE TITLE
PLY mesh format

### ISSUE THREAD

**Issue by dcwhite** [SCIInstitute/SCIRun#426]

**Comment by dcwhite** [comment]
@jessdtate Any knowledge on this one?

**Comment by jessdtate** [comment]
not really. Do you remember who requested this?  I think Mesh lab and paraview have a ply importer exporter.

**Comment by dcwhite** [comment]
I think it was @moritzdannhauer

**Comment by dcwhite** [comment]
**Triage 2026-06:** PLY mesh format support — Moritz originally requested this. MeshLab and ParaView both have PLY support. Implementing an importer would require hooking into the existing Field I/O plugin system (similar to OBJ in #1239). Medium effort. Still relevant for interchange with external mesh tools.
